jenkins-build-collector/src/test/java/com/capitalone/dashboard/collector/HudsonCollectorTaskTests.java

```diff
@@ -94,7 +93,8 @@ public void collect_oneJob_exists_notAdded() {
     @Test
     public void delete_job() {
         HudsonCollector collector = collectorWithOneServer();
-        collector.setId(new ObjectId().get());
+        new ObjectId();
```

Comment:
I fixed this, it is a strange fix done by eclipse, just deleted the new ObjectID(); it is not required.